.gitlab-ci.yml

```diff
@@ -10,7 +10,7 @@ before_script:
 test:
   stage: test
   script:
-    # TODO: run actual tests here when we have them, but for now we just build storybook to test
+    - yarn test
```

Comment:
We should run `yarn test:coverage` instead, and also add the minimum coverage percentages to package.json (see marketplace portal for example). This way we at least keep increasing coverage in the future.